en/pp_limiter_en.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Datenshi Performance Points Limiter"
+old_id: 13
+---
+<div style="text-align:center"><img src="https://user-images.githubusercontent.com/10250068/116959299-c8ae3580-accf-11eb-9ca3-15cd662dbca0.png" /></div>
+
+Hello! this is guide for who got messages from our Bot In-game, we are introduce to you that we have a unique pp limiter, this limiter are mostly prevent abusers/cheater to have more pp, **please note that you are not in banned/restricted mode**. It's just that you won't get more pp, if you play another fresh beatmap you will be get 0pp.
+
+# Limits
+
+These limits are applies both in Relax & Vanilla
+
+- Total PP
+  - 5000pp (STD)
+  - 8888pp (Taiko) 
+  - 8888pp (CTB)
+  - 9999pp (mania)
+- Per Score/Individual PP
+  - Not so disclosed value.
+
+\* PP limit may be changed in future without any notice depending on staff's discretion.
+
+# How to remove the limiter?
```

Comment:
```suggestion
# How to remove the limiter
```
gk pake question mark
kalau mau pake question mark bisa ganti jadi 'How do you remove the limiter?'